app/gui/view/graph-editor/src/component/node/input/widget/single_choice.rs

```diff
@@ -197,6 +215,19 @@ impl SpanWidget for Widget {
     }
 }
 
+/// Find an unambiguous top-level nested call expression inside given node and return its AST ID.
```

Comment:
What is "top-level nested call expression"? How call could be both "top-level" and "nested"?

I think I know what it means, but the wording is confusing.

Also, the function not always returns _nested_ calls, as sometimes it returns the ID from the passed node (first if).